openreview/conference/templates/programchairWebfield.js

```diff
@@ -61,9 +64,15 @@ var getInvitationId = function(name, number) {
 }
 
 var getBlindedNotes = function() {
-  return Webfield.getAll('/notes', {
-    invitation: BLIND_SUBMISSION_ID, noDetails: true
-  });
+  if (PC_PAPER_ASSIGNMENT) {
```

Comment:
Can we get the tags separately from the notes? Each note may have more than one tag from other invitation, and it can be too expensive the call. 

Also, you could do the get tags call only if the tag invitation exists.

Reply:
Agreed. I have made the changes for this.